pkg/blob/blob.go

```diff
@@ -264,7 +266,7 @@ func (d *Driver) GetAuthEnv(ctx context.Context, volumeID, protocol string, attr
 
 	if protocol == nfs {
 		// nfs protocol does not need account key, return directly
-		return accountName, containerName, authEnv, err
+		return accountName, containerName, authEnv, accountKey, err
```

Comment:
return `authEnv` is enough, it already contains `accountKey`

Reply:
done